### PR TITLE
Switch to separated Deflect library

### PR DESCRIPTION
**Pull request by tribal-tec**

### CODE REVIEW

File: eq/client/dc/proxy.h [hunk 1]
```diff
@@ -63,7 +63,7 @@ class Proxy : public boost::noncopyable
     void stopRunning();
 
     /** @return the latest window Event. @sa hasNewEvent() */
-    ::dc::Event getEvent() const;
+    ::deflect::Event getEvent() const;
```
Comment (eile) [inline eq/client/dc/proxy.h]: I guess the :: before deflect is no longer needed?

**Comment by eile** [comment]
+1

**Comment by tribal-tec** [comment]
Retest please

**Comment by tribal-tec** [comment]
Retest this please

**Comment by tribal-tec** [comment]
Retest this please

**Pull request merged**